packages/augur-ui/src/modules/trading/components/form.styles.less

```diff
@@ -494,10 +532,10 @@
 
   align-items: center;
   background-color: var(--color-secondary-action);
-  border: @size-1 solid var(--color-secondary-action);
   display: flex;
   flex-direction: row;
   border-radius: @border-radius-default;
+  border: @size-1 solid var(--color-secondary-action);
```

Comment:
put this back between background-color and display please

Reply:
Updated. Thank you